LEAF_Request_Portal/form.php

```diff
@@ -3170,21 +3170,27 @@ public function query($inQuery)
     public function getDisabledIndicatorList($disabled)
```

Comment:
Type the argument, please.  Not only do we need to get in the habit of this for other reasons, but "$disabled" could really be anything.  And keeps from having to make sure the value is typed before use later in the function.